packages/app-store/hubspotothercalendar/README.mdx

```diff
@@ -6,3 +6,7 @@ items:
 <Slider items={items} />
 
 HubSpot is a cloud-based CRM designed to help align sales and marketing teams, foster sales enablement, boost ROI and optimize your inbound marketing strategy to generate more, qualified leads.
+
+Features:
+ - Creates a meeting in HubSpot for one or many attendees
+ - Attendees that don't exist in HubSpot, gets created first to associate them in the meeting
```

Comment:
Added list of features, the customer that asked for this feature had to ask me what the app provided so to avoid that, here is that info.